frontend/src/components/SignUp.js

```diff
@@ -0,0 +1,46 @@
+import React, {useState} from 'react'
+import './signup.css'
+
+const URL = 'https://project-auth-app.herokuapp.com/users'
+
+export const SignUp = (props) => {
+  const [name, setName] = useState('')
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const [signedUp, setSignedUp] = useState(false)
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+
+    fetch(URL, {
+      method: 'POST',
+      body: JSON.stringify({name: name, email: email, password: password}),
+      headers: {'Content-Type': 'application/json'}
+    }).then(() => {
+      setSignedUp(true)
+      setName('')
+      setEmail('')
+      setPassword('')
```

Comment:
What happens now?  Should we go to the Profile page or do anything with the json response?